File: 프로그래머스/1/12901. 2016년/2016년.java
class Solution {
    int[] MONTH = new int[]{31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
    String[] DAY = new String[]{"THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"};

    public String solution(int a, int b) {
        int count = 0;
        for (int i = 1; i < a; i++) {
            count += MONTH[i - 1];
        };
        count += b; 
        return count % 7 < 0? DAY[DAY.length + (count % 7)] : DAY[count % 7];
    }
}

File: 프로그래머스/1/12901. 2016년/2016년.js
const MONTH = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
const DAY = ["THU", "FRI", "SAT", "SUN", "MON", "TUE", "WED"];


function solution(a, b) {
  let count = 0;
  for (let i = 1; i < a; i++) {
    count += MONTH[i - 1];
  }
    count += b; 
  return count % 7 < 0? DAY[DAY.length + (count % 7)] : DAY[count % 7]
}